usr.bin/netstat/netstat.1

```diff
@@ -401,6 +405,36 @@ See
 .El
 .It Xo
 .Bk -words
+.Nm netstat
+.Fl o
+.Fl 4 | Fl 6
+.Ek
+.Xc
+Print nexthop (nhops) information associated with routing entries.
+When used with
+.Fl 4
+or
+.Fl 6 ,
+limit the output to IPv4 or IPv6 routes respectively.
+Either -4 (IPv4) or -6(IPv6) should be set. 
```

Comment:
I'm not sure this sentence is really necessary; presumably this is like the RFC "should" rather than trying to express that one of -[46] are mandatory?  I'm not sure that we usually make these kinds of recommendations on output filtering once we've described how the filtering works (the sentence before).  If we keep it, these should probably be converted to `.Fl 4` and `.Fl 6` and the spacing made consistent between the two.

Ditto below for -O.

Reply:
Ah, sorry, same comment was made before; I guess I don't like github's diff view.